packages/ses/src/tame-function-constructors.js

```diff
@@ -49,7 +49,7 @@ export default function tameFunctionConstructors() {
     FERAL_FUNCTION.prototype.constructor('return 1');
   } catch (ignore) {
     // Throws, no need to patch.
-    return {};
+    return harden({});
```

Comment:
This is wrong. It crashes in our environment.

![image](https://user-images.githubusercontent.com/5390719/189535478-707a6938-c148-40a9-8537-398d22280b1c.png)

Because when this function is called, there is no `harden` defined yet.